From d580a058d84196bd7017fcabd9ae289eff68b32b Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Mon, 17 Jun 2019 23:46:21 -0700
Subject: [PATCH 1445/1548] dma-buf/hyper_dmabuf: remove unnecessary check and
 debug log

Removed unnecessary check on the dmabuf whether it has been changed
in between. Also, removed some redundant debug print out.

Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c b/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c
index 5c7101ecc742..ee499294fc15 100644
--- a/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c
+++ b/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c
@@ -223,19 +223,7 @@ static void hyper_dmabuf_ops_release(struct dma_buf *dma_buf)
 		return;
 	}
 
-	dev_dbg(hy_drv_priv->dev, "%s: {%x,%x} dmabuf:%p ref_c:%d\n", __func__,
-			imported->hid.id, imported->hid.rng_key[0],
-			imported->dma_buf, imported->importers);
-
-	if (dma_buf != imported->dma_buf) {
-		dev_dbg(hy_drv_priv->dev, "%s: dma_buf changed!\n", __func__);
-		mutex_unlock(&hy_drv_priv->lock);
-		return;
-	}
-
-	dev_dbg(hy_drv_priv->dev, "%s: clear imported->dma_buf\n", __func__);
 	imported->dma_buf = NULL;
-
 	imported->importers--;
 
 	if (imported->importers == 0) {
@@ -257,7 +245,6 @@ static void hyper_dmabuf_ops_release(struct dma_buf *dma_buf)
 			__func__, finish, imported->importers,
 			imported->valid ? 'Y':'N');
 
-
 	/* release operation should be synchronized with exporter. */
 	sync_request(imported->hid, HYPER_DMABUF_OPS_RELEASE, true);
 
-- 
2.20.1

